apps/core/scenes/resultDetail/resultDetailContent/ResultDetailPassenger.js

```diff
@@ -36,6 +38,22 @@ class ResultDetailPassenger extends React.Component<Props, State> {
     if (!(validatingPassenger.passportId && validatingPassenger.name)) {
       return 'ID and Name are mandatory';
     }
+    console.log(
```

Comment:
`console.log` for testing purposes only plz 🙏

Reply:
Dont worry I ll remove it, the condition is not working properly yet, so I kept it